src/main/drivers/bus_spi.c

```diff
@@ -101,6 +101,20 @@ bool spiInit(SPIDevice device)
         return true;
 #else
         break;
+#endif
+    case SPIDEV_5:
+#if defined(USE_SPI_DEVICE_4)
```

Comment:
Logic sense tingling: Should this be `USE_SPI_DEVICE_5`? (And 6 below.)

Reply:
Probably ... 
Using bitmask for `USE_SPI_DEVICE` and macro tricks from time will make this code trivial ... But the macro magic under it is a bit nontrivial